pkg/http-server/start.go

```diff
@@ -54,14 +56,20 @@ func (g *APIServer) start(routes []*Route, port, certFile, privateKeyFile string
 		router = mux.NewRouter()            // new router
 	)
 
+	logWriter := logging.GetLogWriter(logger)
+
 	logger.Info("registering routes...")
 
 	// register all routes
 	for _, v := range routes {
 		logger.Info("Route ", v.verb, " - ", v.path)
-		router.Methods(v.verb).Path(v.path).HandlerFunc(v.fn)
+		handler := gorillaHandlers.CustomLoggingHandler(logWriter, http.HandlerFunc(v.fn), logging.WriteRequestLog)
+		router.Methods(v.verb).Path(v.path).Handler(handler)
 	}
 
+	router.NotFoundHandler = gorillaHandlers.CustomLoggingHandler(logWriter, http.HandlerFunc(httputils.NotFound), logging.WriteRequestLog)
```

Comment:
was the sole purpose of using a custom logging handler to remove the timestamp that is logged by the default log formatter provided by gorilla handlers?

Reply:
nope. its more than that. its apache standard http server log format.

Reply:
The reason I asked is because, it seems to be the only difference between the default log formatter and the log formatter you have implemented. Can you please check once?
https://github.com/gorilla/handlers/blob/master/logging.go#L144